Remotion/Data/DomainObjects/Queries/Configuration/QueryTypeExtensions.cs

```diff
@@ -0,0 +1,37 @@
+// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+// 
+// The re-motion Core Framework is free software; you can redistribute it 
+// and/or modify it under the terms of the GNU Lesser General Public License 
+// as published by the Free Software Foundation; either version 2.1 of the 
+// License, or (at your option) any later version.
+// 
+// re-motion is distributed in the hope that it will be useful, 
+// but WITHOUT ANY WARRANTY; without even the implied warranty of 
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
+// GNU Lesser General Public License for more details.
+// 
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+// 
+using Remotion.Utilities;
+
+namespace Remotion.Data.DomainObjects.Queries.Configuration;
+
+/// <summary>
+/// Provides convenience extension methods for <see cref="QueryType"/>.
+/// </summary>
+public static class QueryTypeExtensions
+{
+  public static bool IsReadWrite (this QueryType queryType)
```

Comment:
I don't think we need this as a public method. it can be a simple "IsWriteAccessMandatory" private method on RootPersistenceStrategy. 

Why not public: it suggests that evaulating this is soeemthing that's a good idea/helpful when there should not be a reason to do this manually/ourside of the infrastucture.